front/src/App.js

```diff
@@ -156,13 +146,24 @@ class App extends Component {
    * @param onIncorrectToken Callback from AuthenticationScreen to update the
    * AuthenticationScreen in case the token was incorrect.
    */
-  onSubmit(token, onIncorrectToken) {
-    //TODO: Implement a correct version
-    if (token === "techunited") {
-      this.setState({ loggedIn: true, token: token });
-    } else {
-      onIncorrectToken();
-    }
+  AuthSubmit(token, onIncorrectToken) {
```

Comment:
function naming guidelines probably dictate that function names need to be camelCased.